Lib/test/test_warnings/__init__.py

```diff
@@ -899,6 +899,15 @@ def test_showwarning(self):
         self.module.showwarning(message, category, file_name, line_num,
                                 file_object, expected_file_line)
         self.assertEqual(expect, file_object.getvalue())
+        # Test that file is flushed after writing the warning.
+        raw = BytesIO()
+        file_object = TextIOWrapper(BufferedWriter(raw), 'utf-8',
```

Comment:
```suggestion
        file_object = TextIOWrapper(BufferedWriter(raw))
```

Reply:
As far as I understand, omitting the encoding is soft deprecated (PEP 597).

Reply:
Yes sorry.